src/WebJobs.Script/Workers/Http/HttpScriptInvocationResultExtensions.cs

```diff
@@ -35,6 +35,11 @@ public static ScriptInvocationResult ToScriptInvocationResult(this HttpScriptInv
                 {
                     scriptInvocationResult.Return = GetBindingValue(returnParameterBindingMetadata.DataType, httpScriptInvocationResult.ReturnValue);
                 }
+                else
+                {
+                    // Some triggers (like Durable Functions' triggers) assign $return implicitly.
+                    scriptInvocationResult.Return = httpScriptInvocationResult.ReturnValue;
```

Comment:
Prior to this change, return values were only saved when there was an explicit `$return` binding declared in function.json. Durable Functions triggers, however, do not use explicit `$return` bindings. Instead, the `$return` binding is implicit in the trigger implementation, thus the return value wasn't being returned by this logic.

I couldn't think of any meaningful reason why return values should be ignored when there's no `$return` binding in function.json, so I added this `else` case to unblock Durable Functions, and any other trigger that might also have implicit return value bindings.